Makefile

```diff
@@ -415,6 +415,8 @@ FZ_MODULES = \
 			$(MOD_LOCK_FREE) \
 			$(MOD_NOM)
 
+C_FILES = $(shell find $(FZ_SRC) \( -path ./build -o -path ./.git \) -prune -o -name '*.c' -print)
```

Comment:
Are you sure about that?

```
~/fuzion
❯ find build -type f -name "*.c" 
build/include/posix.c
build/include/win.c
build/include/shared.c

~/fuzion
❯ make lint/c       
clang-tidy ./include/posix.c ./include/win.c ./include/shared.c -- -std=c11
[1/3] Processing file /home/simon/fuzion/include/posix.c.
10 errors generated.
Error while processing /home/simon/fuzion/include/posix.c.
[2/3] Processing file /home/simon/fuzion/include/win.c.
11 errors generated.
Error while processing /home/simon/fuzion/include/win.c.
[3/3] Processing file /home/simon/fuzion/include/shared.c.
3 warnings and 11 errors generated.
…

~/fuzion
❯ cd build

~/fuzion/build
❯ make lint/c -C .. 
make: Entering directory '/home/simon/fuzion'
clang-tidy ./include/posix.c ./include/win.c ./include/shared.c -- -std=c11
[1/3] Processing file /home/simon/fuzion/include/posix.c.
10 errors generated.
Error while processing /home/simon/fuzion/include/posix.c.
[2/3] Processing file /home/simon/fuzion/include/win.c.
11 errors generated.
Error while processing /home/simon/fuzion/include/win.c.
[3/3] Processing file /home/simon/fuzion/include/shared.c.
3 warnings and 11 errors generated.
Error while processing /home/simon/fuzion/include/shared.c.
…
```

Reply:
Without the option `-C` you can not run make for fuzion from that location you run the `find` command but `make lint/c -C ~/src/fuzion` does work. So it does not convince me that that alone is a problem.

As for other c files that should not be checked I did not think about that. My Idea was to have this still work in case there might be c files in different locations in the future.
But if you still think it is better/safer to just check one location or if it is unlikely that there will be c files in other locations I will change it.